### PR TITLE
chore: upgrade kapa to 1.6.1

### PR DESCRIPTION
**Pull request by docmerlin**

### CODE REVIEW

**Comment by prashanthjbabu** [comment]
@yosifkit  we're waiting on kapacitor 1.6.1 image for our deployments . Can you please prioritize this?

**Comment by tianon** [comment]
Now that the key loop only has one key and one server, it could definitely be simplified (in a future PR). :sweat_smile:

File: library/kapacitor [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Maintainers: Jonathan A. Sternberg <jonathan@influxdata.com> (@jsternberg),
              j. Emrys Landivar <landivar@gmail.com> (@docmerlin)
 GitRepo: git://github.com/influxdata/influxdata-docker
-GitCommit: 63fe0684150d96916514a84cf5978c3512dc15fa
+GitCommit: 0e03b615c48d7f089fff7bb2aa2e7d65e83bd2c2
 
 Tags: 1.4, 1.4.1
```
Comment (tianon) [inline library/kapacitor:6]: Just want to double check that 1.4.x is still actively supported (and not EOL now that 1.6.x is released).

As a reminder, removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

File: library/kapacitor [hunk 2]
```diff
@@ -10,9 +10,17 @@ Directory: kapacitor/1.4
...
+Tags: 1.5, 1.5.9
 Architectures: amd64, arm32v7, arm64v8
 Directory: kapacitor/1.5
 
-Tags: 1.5-alpine, 1.5.9-alpine, alpine
+Tags: 1.5-alpine, 1.5.9-alpine
 Directory: kapacitor/1.5/alpine
+
+Tags: 1.6, 1.6.1, latest
+Architectures: amd64, arm32v7, arm64v8
```
Comment (tianon) [inline library/kapacitor:21]: This should probably be minus `arm32v7`, right?  (looks like `armhf` was dropped from the relevant `case` statement in this version)

Comment (docmerlin) [reply library/kapacitor:21]: thanks.

**Comment by tianon** [comment]
Friendly ping?

**Comment by docmerlin** [comment]
> Friendly ping?

sorry, I was out on vacation last week.  I have made the changes suggested.  Thanks.

**Pull request merged**